downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -252,10 +257,9 @@ abstract class VersionControlSystem {
     abstract fun getWorkingTree(vcsDirectory: File): WorkingTree
 
     /**
-     * Return true if the type name matches this VCS. For example for SVN it should return true on "svn",
-     * "subversion", or any other spelling that clearly identifies SVN.
+     * Return true if any of the [names] matches [vcsType]. Comparison is done case-insensitive.
```

Comment:
"case-insensitively"

Reply:
How do you guarantee comparison to be done case-insensitively? Is it a convention now that both `vcsType` and `names` have to be lower case? Otherwise you probably need something like

    names.any { it.toLowerCase() == vcsType.toLowerCase() }

Reply:
It's a convention documented in the `aliases` property. Everything done.